src/main/java/com/google/devtools/build/lib/rules/apple/ApplePlatform.java

```diff
@@ -47,9 +47,9 @@ public enum ApplePlatform implements ApplePlatformApi {
   private static final ImmutableSet<String> IOS_DEVICE_TARGET_CPUS =
       ImmutableSet.of("ios_armv6", "ios_arm64", "ios_armv7", "ios_armv7s", "ios_arm64e");
   private static final ImmutableSet<String> WATCHOS_SIMULATOR_TARGET_CPUS =
-      ImmutableSet.of("watchos_i386", "watchos_x86_64", "watchos_arm64");
+      ImmutableSet.of("watchos_i386", "watchos_x86_64", "watchos_sim_arm64");
   private static final ImmutableSet<String> WATCHOS_DEVICE_TARGET_CPUS =
-      ImmutableSet.of("watchos_armv7k", "watchos_arm64_32");
+      ImmutableSet.of("watchos_armv7k", "watchos_arm64_32", "watchos_arm64");
```

Comment:
Since this is a breaking change, we should add something to the release notes at the minimum, though maybe we don't change the name at all? People can have existing configuration that uses `watchos_arm64` to mean the simulator, and some rulesets would have to do a version check or something similar to get the right version of this: https://github.com/buildbuddy-io/rules_xcodeproj/blob/47d9510b64878ae1893d355cbf134d6a4b6eb715/xcodeproj/internal/xcodeproj_rule.bzl#L490-L493

Reply:
> Since this is a breaking change, we should add something to the release notes at the minimum, though maybe we don't change the name at all?

I think breaking change would be suitable for 6.0.0 release? 👀 I wanted to be consistent with the other names, otherwise it can get confusing.

Reply:
It's the best time to break it, yes, but I was also starting how it's tricky for rulesets to do things based on the bazel version.